packages/editor/src/components/url-popover/index.js

```diff
@@ -0,0 +1,69 @@
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import {
+	Popover,
+	IconButton,
+} from '@wordpress/components';
+
+class URLPopover extends Component {
```

Comment:
I had a couple of quandries about this new component.
1. Should it be in the components package instead of editor?
2. Technically, this could be used to edit anything. As such, I could go as far to call it something like InlineEditor, EditPopover, InlineEditPopover. 🤷‍♂️

Reply:
I think you made the right call!